plugins/chat/assets/javascripts/discourse/components/chat/header/icon.gjs

```diff
@@ -61,10 +62,6 @@ export default class ChatHeaderIcon extends Component {
   }
 
   get href() {
-    if (this.site.mobileView && this.chatStateManager.isFullPageActive) {
-      return getURL("/chat");
-    }
-
```

Comment:
since we do not show the icon on mobile view, there is no need to compute this case.